pkg/crdupdater/namespace.go

```diff
@@ -89,5 +108,24 @@ func (n *Namespace) remove() error {
 
 // patch patches the namespace
 func (n *Namespace) patch(ns interface{}, isPatched bool) (bool, error) {
+	namespace := ns.(*corev1.Namespace)
+	namespace.Labels = util.InitLabels(namespace.Labels)
+	var app, name string
+	if appVal, ok := n.config.expectedLabels["app"]; ok {
+		app = appVal.value[0]
+	}
+	if nameVal, ok := n.config.expectedLabels["name"]; ok {
+		name = nameVal.value[0]
+	}
+	if val, ok := namespace.Labels[fmt.Sprintf("synopsys.com/%s.%s", app, name)]; !ok || val != n.config.version {
+		if len(app) > 0 && len(name) > 0 && len(n.config.version) > 0 {
```

Comment:
If this case is false we probably should log an error with the values we wanted to use.  This failure seems like something that shouldn't happen, but if did we'd want to know the data it was trying to use

Reply:
done